build

```diff
@@ -1,54 +1,63 @@
 #!/bin/bash
-cd /tmp/rom # Depends on where source got synced
+cd /tmp/rom || exit                               # Depends on where source got synced
+#--------------------------EDIT FROM HERE-------------------------------
+#  EDIT IMP VARIABLES AND COLLECT CCACHE BEFORE BUILD ELSE CI TIMEOUT
 
-# Normal build steps
-. build/envsetup.sh
-lunch aosp_mido-user
-export CCACHE_DIR=/tmp/ccache
-export CCACHE_EXEC=$(which ccache)
-export USE_CCACHE=1
-ccache -M 20G # It took only 6.4GB for mido
-ccache -o compression=true # Will save times and data to download and upload ccache, also negligible performance issue
-ccache -z # Clear old stats, so monitor script will provide real ccache statistics
-
-# Next 8 lines should be run first to collect ccache and then upload, after doning it 1 or 2 times, our ccache will help to build without these 8 lines
-#make api-stubs-docs || echo no problem, we need ccache
-#make hiddenapi-lists-docs || echo no problem, we need ccache
-#make system-api-stubs-docs || echo no problem we need ccache
-#make test-api-stubs-docs || echo no problem, we need ccache
-#make aex -j10 & # dont remove that '&'
-#sleep 85m
-#kill %1
-#ccache -s
-#and dont use below codes for first 1 or 2 times, to get ccache uploaded,
-
-
-# upload function for uploading rom zip file! I don't want unwanted builds in my google drive haha!
+export botapi="$mybot"  # add your own bot api                      
```

Comment:
hiding bot API is better, who knows how to hide rclone_config, and use rclone config, he can hide bot API too if he wants.